.github/workflows/release.yaml

```diff
@@ -51,9 +51,14 @@ jobs:
 
       - name: Push Helm Chart to OCI Registries
         run: |
+          shopt -s nullglob
           REPO_OWNER=`echo "${{ github.repository_owner }}" | tr '[:upper:]' '[:lower:]'`
           # Get packed chart file name
-          PKG_NAME=`ls .cr-release-packages/*.tgz`
-          # pushing chart to OCI registries
-          helm push ${PKG_NAME} oci://registry-1.docker.io/cartesi
-          helm push ${PKG_NAME} oci://ghcr.io/${REPO_OWNER}/charts
+          PKG_NAME=(.cr-release-packages/*.tgz)
```

Comment:
can't we run this on a previous step, save its output, and skipt this step using `step.if: ` clause?

this way, the CI execution will be more clear that that step was skiped, instead of "Push Helm Chart to OCI Registries" :heavy_check_mark: 

Got it?

Reply:
We can't use script in `helm/chart-releaser-action@v1.5.0` actions. so we have two ways : 

1 - Do the way I wrote 

2 - Add another step to check the file 

Which one is better ? ( I'd prefer the first one because the second one increases our code)

Reply:
I prefer the 2, since we'll have a more clear output for the job execution.

Reply:
Fixed !